prisma/schema.prisma

```diff
@@ -0,0 +1,17 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
```

Comment:
When we do deployments we should check if there is a way in Azure to set up environment variables because I believe the .env file is excluded in the gitignore